ios/ApplePay.swift

```diff
@@ -53,16 +53,21 @@ class ApplePay: UIViewController {
 extension ApplePay: PKPaymentAuthorizationViewControllerDelegate {
     func paymentAuthorizationViewControllerDidFinish(_ controller: PKPaymentAuthorizationViewController) {
         controller.dismiss(animated: true, completion: nil)
+        if (self.resolve != nil) {
+          self.resolve!("PAYMENT_CANCELLED")
+        }
     }
```

Comment:
when apple pay successful and apay view is closing, this method triggered too...

Reply:
Yes it is, if you look at the successful handler, self.resolve is set to nil after being called, this prevents ` self.resolve!("PAYMENT_CANCELLED")` from being called.
When successful the handler is called first before `paymentAuthorizationViewControllerDidFinish`